.github/labeler.yml

```diff
@@ -54,9 +54,16 @@ docker:
   - 'inlong-dataproxy/dataproxy-docker/**/*'
   - 'inlong-manager/manager-docker/**/*'
   - 'inlong-tubemq/tubemq-docker/**/*'
-  - '!inlong-tubemq/tubemq-docker/tubemq-k8s/**/*'
```

Comment:
@dockerzhang This configuration item can cause the `docker` label to be incorrectly labeled, so I removed it.